lib/Client.js

```diff
@@ -1042,31 +1054,104 @@ class Client extends EventEmitter {
         }).then((instance) => new StageInstance(instance, this));
     }
 
+    /**
+     * Create a thread in a channel
+     * @arg {String} channelID The ID of the channel
+     * @arg {Object} [options] The thread options
+     * @arg {Array<String>} [options.appliedTags] The IDs of the set of tags that have been applied to a thread in a forum channel (threads created in forum channels only, max 5)
+     * @arg {Number} [options.autoArchiveDuration] The duration in minutes to automatically archive the thread after recent activity, either 60, 1440, 4320 or 10080 (thread channels only)
+     * @arg {Boolean} [options.invitable] Whether non-moderators can add other non-moderators to the thread (private threads only)
+     * @arg {Object} [options.message] The message to create with the thread (only use when creating a thread inside of a `GUILD_FORUM` channel). Note: When creating a forum channel thread, you must provide at least one of `content`, `embeds`, `stickerIDs`, `components`, or `files`
```

Comment:
Would rather just link to discord docs and specify camelCase allowedMentions and stickerIDs to be completely honest

Reply:
If there other cases like this, would it not make more sense to fix them all within the same PR? 👍🏻